TestApps/Samples/Samples/TreeViews.cs

```diff
@@ -168,6 +168,16 @@ public TreeViews ()
 				Console.WriteLine("Collapsed: " + val);
 			};
 
+			RadioButtonGroup group = new RadioButtonGroup ();
```

Comment:
Don't we need to set an active one?

Reply:
No, the toolkits usually always activate the first option in a group by default.